class.SecurePage.php

```diff
@@ -5,6 +5,8 @@
 class SecurePage extends FlipPage
 {
     public $secure_root;
+    public $secureUrl;
```

Comment:
This is a redundant definition now right? This is defined in FlipPage in Common per your pull.